Sources/PostgresNIO/New/Connection State Machine/ExtendedQueryStateMachine.swift

```diff
@@ -319,7 +323,23 @@ struct ExtendedQueryStateMachine {
     }
     
     mutating func emptyQueryResponseReceived() -> Action {
-        preconditionFailure("Unimplemented")
+        guard case .bindCompleteReceived(let queryContext) = self.state else {
+            return self.setAndFireError(.unexpectedBackendMessage(.emptyQueryResponse))
+        }
+
+        switch queryContext.query {
+        case .unnamed(_, let eventLoopPromise),
+                .executeStatement(_, let eventLoopPromise):
```

Comment:
```suggestion
            .executeStatement(_, let eventLoopPromise):
```

Reply:
Hmm i agree what you mentioned is better, but what is in the PR is what Xcode indents the line to be.
Do you want to go against that?